Source/Building_BedMedPod.cs

```diff
@@ -604,7 +604,7 @@ public override void Tick()
             {
                 DiagnosingTicks--;
                 powerComp.PowerOutput = -DiagnosingPowerConsumption;
-                PatientPawn.needs.food.CurLevelPercentage = 1f;
+                if (PatientPawn != null) PatientPawn.needs.food.CurLevelPercentage = 1f;
```

Comment:
Good catch on the nullcheck for `PatientPawn`.

I'll add the braces myself after merging this PR, as omitting braces for one-line if statements is generally not good practice (detracts from code readability, and may cause problems if someone wants to stick additional code inside the `if{}` block later)